panel/models/widgets.py

```diff
@@ -127,3 +127,22 @@ class FileDownload(InputWidget):
     """)
 
     title = Override(default='')
+
+
+class SliderInput(Slider):
+    """
+    Slider associate with an input
+    """
+    tooltips = Override(default=False)
+
+    hard_start = Float(default=None, help="""
+    The minimum allowable value accept by the input.
+    """)
+
+    hard_end = Float(default=None, help="""
+    The maximum allowable value accept by the input.
+    """)
+
+    input_size = String(default=None, help="""
+    Size of the input in px or %
```

Comment:
```suggestion
    Size of the widget in px or %
```
(Not sure about the name; seems like this is widget_size?  By input I'd expect to refer to the numeric value being put in, not the graphical widget.)

Reply:
Actualy it's really just the size of the numeric input
the size of the overall widget is define by width

Reply:
Ah! I consider both the slider and the text box to be ways of providing input, and so referring to one part of it as input confused me. :-) 

Maybe `slider_size` (if needed) and `textbox_size`?